backend/wmg/data/validation/validation.py

```diff
@@ -0,0 +1,384 @@
+        )
+        AQP5_non_high_expression_cell_types = AQP5_human_lung_cube.query(
+            f"cell_type_ontology_term_id not in {AQP5_high_expression_cell_type_ids}"
+        )
+        AQP5_high_expression_avg = (
+            AQP5_high_expression_cell_types.sum()["sum"] / AQP5_high_expression_cell_types.sum()["nnz"]
+        )
+        AQP5_non_high_expression_avg = (
+            AQP5_non_high_expression_cell_types.sum()["sum"] / AQP5_non_high_expression_cell_types.sum()["nnz"]
+        )
+        if AQP5_non_high_expression_avg > AQP5_high_expression_avg:
+            self.errors.append("AQP5 expression levels are off")
+
+    def validate_expression_levels_for_particular_gene_dataset(self):
+        human_ont_id = fixtures.validation_species_ontologies["human"]
+        human_lung_int = fixtures.validation_tissues_with_many_cell_types["lung"]
+        MALAT1_ont_id = fixtures.validation_gene_ontologies["MALAT1"]
+        CCL5_ont_id = fixtures.validation_gene_ontologies["CCL5"]
+        with tiledb.open(self.expression_summary_path) as cube:
+            MALAT1_human_lung_cube = cube.df[
+                MALAT1_ont_id:MALAT1_ont_id, human_lung_int:human_lung_int, human_ont_id:human_ont_id
+            ]
+            CCL5_human_lung_cube = cube.df[
+                CCL5_ont_id:CCL5_ont_id, human_lung_int:human_lung_int, human_ont_id:human_ont_id
+            ]
+
+            MALAT1_expression = MALAT1_human_lung_cube.query(f"dataset_id == '{self.validation_dataset_uuid}'")
+            CCL5_expression = CCL5_human_lung_cube.query(f"dataset_id == '{self.validation_dataset_uuid}'")
+
+            malat1_expression_sum_by_cell_type = MALAT1_expression.groupby("cell_type_ontology_term_id").sum()["sum"]
+            ccl5_expression_sum_by_cell_type = CCL5_expression.groupby("cell_type_ontology_term_id").sum()["sum"]
+
+            expected_values = anndata.read_h5ad(
+                f"{pathlib.Path(__file__).parent.resolve()}/lung_map_3de0ad6d-4378-4f62-b37b-ec0b75a50d94.h5ad"
+            )
+            malat_expected = expected_values.obs.assign(MALAT1=expected_values.layers["rankit"].toarray()[:, 0])
+            ccl5_expected = expected_values.obs.assign(CCL5=expected_values.layers["rankit"].toarray()[:, 1])
+
+            expected_malat1_by_cell_type = malat_expected.groupby("cell_type_ontology_term_id").sum().MALAT1
+            expected_ccl5_by_cell_type = ccl5_expected.groupby("cell_type_ontology_term_id").sum().CCL5
+            # drop ccl5 cell types with expression value of zero (to match pipeline processing)
+            expected_ccl5_by_cell_type = expected_ccl5_by_cell_type[expected_ccl5_by_cell_type != 0]
+
+            malat1_comparison = expected_malat1_by_cell_type.compare(malat1_expression_sum_by_cell_type)
+            ccl5_comparison = expected_ccl5_by_cell_type.compare(ccl5_expression_sum_by_cell_type)
+            logger.info(malat1_comparison)
+            logger.info(ccl5_comparison)
+
+            """
```

Comment:
@pablo-gar can you take a look at 357 - 374 and just confirm it seems reasonable?

Reply:
looks good to me! less than 1 difference is very strict and I'm glad the method is working to this extent